tests/csip_test.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021-2023 Google LLC
```

Comment:
I put `csip_test.py` under `test/` because `test_utils.py` is not accessible from a subdirectory.
Might be a better solution?

Reply:
That sounds fine to me.